### PR TITLE
OKRS24-24 

### PR DESCRIPTION
**Pull request by dmytrotsko**
Combined covidcast and flu, etc. sections.

### CODE REVIEW

**Comment by melange396** [comment]
Update: @dmytrotsko was looking into adding [aliases](https://gohugo.io/content-management/urls/#aliases) for the new paths, so old URLs will still work.  We should keep the aliases around at least until search engines recrawl our site and update the links in their indexes.

**Comment by nmdefries** [comment]
Approved by Roni

File: themes/delphi/layouts/partials/footer.html [hunk 1]
```diff
@@ -9,7 +9,7 @@ <h5 class="uk-text-bold">COVIDcast</h5>
         <ul class="uk-list">
```
Comment (dshemetov) [inline themes/delphi/layouts/partials/footer.html:9]: The h5 element above this still refers to COVIDcast, should probably be Epidemic Signals

**Review by dshemetov — APPROVED** [review]
COVID and Flu & Other Pathogens are now combined in one section as requested in [OKRS24-24](https://cmu-delphi.atlassian.net/browse/OKRS24-24). The `/epidemic-signals/_index.md` also has the alias `covid19`, for backwards compatibility. Footer looks good.

**Pull request merged**